src/muse/commodities.py

```diff
@@ -66,17 +66,9 @@ def just_tech(x):
                 dims.remove("commodity")
             return x.any(dims)
 
-        products = list(
-            just_tech(technologies[x] > 0)
-            for x in {"fixed_outputs", "flexible_outputs"}
-            if x in technologies.data_vars
-        )
-        if len(products) == 0:
+        if "fixed_outputs" not in technologies.data_vars:
             raise ValueError("Missing output array in technologies")
```

Comment:
Maybe indicate what array is the one missing:
```suggestion
            raise ValueError("Missing 'fixed_outputs' array in technologies")
```
Or something along these lines?